src/rtp.c

```diff
@@ -365,6 +365,10 @@ int janus_rtp_header_extension_set_abs_send_time(char *buf, int len, int id, uin
 	return 0;
 }
 
+/*
+	It parses only the shortened version of abs-capture-time without estimated-capture-clock-offset
+	Check http://www.webrtc.org/experiments/rtp-hdrext/abs-capture-time for details.
+*/
```

Comment:
This comment is unneeded here. The other functions don't have such a description either. The description is already available in the header.

Reply:
Removed